core/src/main/java/org/apache/spark/shuffle/checksum/ShuffleChecksumHelper.java

```diff
@@ -0,0 +1,83 @@
+package org.apache.spark.shuffle.checksum;
```

Comment:
No license header.

Reply:
oh yeah, this's definately a mistake. Though, it doesn't seem to be related to `RAT` failure.

Reply:
RAT is a license checker.

> https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/140992/console

```
========================================================================
Running Apache RAT checks
========================================================================
Attempting to fetch rat
Could not find Apache license headers in the following files:
 !????? /home/jenkins/workspace/SparkPullRequestBuilder/core/src/main/java/org/apache/spark/shuffle/checksum/ShuffleChecksumHelper.java
[error] running /home/jenkins/workspace/SparkPullRequestBuilder/dev/check-license ; received return code 1
```

Reply:
Oh I see. I was caught by the `FileNotFoundException` previously.